db/seeds.rb

```diff
@@ -5,3 +5,16 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+
+
+array = Array.new
+
+100.times do
+  hash = Hash.new
+  hash[:title] = Faker::Name.title
+  hash[:body] = Faker::Lorem.paragraphs(rand(5)+1).join(" ")
+  hash[:user_id] = User.first
```

Comment:
this is a seed file, this should be ran under the assumption that the db is clean and empty. dont use `User.first`, unless you create a user first. If I cloned this repo and ran the seed, it would give me errors. You should create a few users first and then use `User.all.sample` for the `:user_id`